tests/defer/src/appleTest/kotlin/test/DeferAppleStreamingTest.kt

```diff
@@ -0,0 +1,58 @@
+package test
+
+import com.apollographql.apollo3.ApolloClient
+import com.apollographql.apollo3.annotations.ApolloExperimental
+import com.apollographql.apollo3.mockserver.MockServer
+import com.apollographql.apollo3.mockserver.enqueueMultipart
+import com.apollographql.apollo3.mpp.currentTimeMillis
+import com.apollographql.apollo3.testing.runTest
+import defer.WithFragmentSpreadsQuery
+import kotlinx.coroutines.flow.collect
+import kotlin.test.Test
+import kotlin.test.assertTrue
+
+// TODO Move this test to the common test suite when chunked support works on JS
```

Comment:
You could move this code to commonTest now and return early if the test is running on JS. We have this in a few other places like https://github.com/apollographql/apollo-kotlin/blob/c4c605e84cb1006d8b81936886cbaed49ae0d262/tests/models-response-based/src/commonTest/kotlin/test/ParseResponseBodyTest.kt#L60